_config.yml

```
@@ -98,3 +98,18 @@ defaults:
     values:
       layout: single
       author_profile: false
+  # _faq
+  - scope:
+      path: ""
+      type: faq
+    values:
+      layout: single
+      author_profile: false
+      share: true
```

```suggestion
      share: false
```
as suggested by @TeriForey in https://github.com/RSE-leaders/SORSE20/pull/64#issuecomment-643940335